site/oc/core.clj

```diff
@@ -4,7 +4,7 @@
             [hiccup.page :as hp]
             [environ.core :refer (env)]))
 
-(def contact-email "hello@opencompany.com")
+(def contact-email "hello@carrot.io")
```

Comment:
Let's keep this as opencompany.com for now. We haven't set up email on carrot.io yet.